src/GraphQL/Execution/ExecutionResult.cs

```diff
@@ -1,26 +1,51 @@
 using System;
 using System.Collections.Generic;
+using GraphQL.Execution;
 using GraphQL.Instrumentation;
 using GraphQL.Language.AST;
 
 namespace GraphQL
 {
     public class ExecutionResult
     {
+        /// <summary>
+        /// Returns the data from the graph resolvers. This property is typically serialized as part of the GraphQL json response.
+        /// </summary>
         public object Data { get; set; }
 
+        /// <summary>
+        /// Returns a set of errors that occurred during processing. This property is typically serialized as part of the GraphQL json response.
+        /// </summary>
         public ExecutionErrors Errors { get; set; }
 
+        /// <summary>
+        /// Returns the original query.
+        /// </summary>
         public string Query { get; set; }
 
+        /// <summary>
+        /// Returns the parsed GraphQL request.
+        /// </summary>
         public Document Document { get; set; }
 
+        /// <summary>
+        /// Returns the GraphQL operation that is being executed.
+        /// </summary>
         public Operation Operation { get; set; }
 
+        /// <summary>
+        /// Returns the performance metrics (Apollo Tracing) when enabled by <see cref="ExecutionOptions.EnableMetrics"/>.
+        /// </summary>
         public PerfRecord[] Perf { get; set; }
 
+        /// <summary>
+        /// Indicates that unhandled <see cref="Exception"/> stack traces should be serialized into GraphQL query result json along with exception messages; otherwise only <see cref="Exception.Message"/> should be serialized
```

Comment:
```suggestion
        /// Indicates that unhandled <see cref="Exception"/> stack traces should be serialized into GraphQL response json along with exception messages; otherwise only <see cref="Exception.Message"/> should be serialized
```